src/main/java/ma/norsys/bookstore/services/BookServiceImp.java

```diff
@@ -0,0 +1,89 @@
+package ma.norsys.bookstore.services;
+
+import ma.norsys.bookstore.exceptions.BookNotFoundException;
+import com.example.evaluation_livre.services.IBookService;
+import ma.norsys.bookstore.exceptions.PostArgumentNotValidException;
+import ma.norsys.bookstore.models.Book;
+import ma.norsys.bookstore.repositories.BookRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+
+@Service
+public class BookServiceImp implements IBookService {
```

Comment:
```suggestion
public class BookService {
```

delete the interface reason:  https://dimitri.codes/spring-interface/